architecture_docs/ADRs/0000-use_architectural_decision_records.md

```diff
@@ -0,0 +1,198 @@
+# Use Architectural Decision Records
+
+# Status
+
+Accepted
+
+## Context & Problem Statement
+
+There are some choices we made in the past where we debated, weighed alternatives and tradeoffs,
+but we didn't document our choices or why we chose them.
+This means that newcomers to the project might not undertand why these choices happened.
+But even longtime contributors can forget!
+
+The benefits of ADRs have been outlined in [Michael Nygard's "Documenting architecture decisions"][nygard-blog].
+There are additional references in [this Red Hat blog post about ADRs](https://www.redhat.com/architect/architecture-decision-records).
+
+Stylistically, there are many formats to consider, each with different tradeoffs. How meta!
+
+## Decision
+
+We will use ADRs. 
+
+ADRs must be for important decisions regarding Pelorus's architecture, not to pretend we are IETF members.
+
+Proposed ADRs will be written up in the below format, and opened as a Pull Request.
+
+### The Format
+
+We will use GitHub Flavored Markdown documents.
+
+The final file name will be a unique sequential number that follows the `NNNN-title_after_dash_with_underscores` pattern.  
```

Comment:
I would change the format, it is a mix of kebab and snake case. I would use the style that is decided in here https://github.com/konveyor/pelorus/issues/817